bears/vcs/git/GitCommitBear.py

```diff
@@ -41,13 +50,39 @@ def get_body_checks_metadata(cls):
             omit={'self', 'body'})
 
     @classmethod
+    def get_issue_checks_metadata(cls):
+        return FunctionMetadata.from_function(
+            cls.check_issue,
+            omit={'self', 'body'})
+
+    @classmethod
     def get_metadata(cls):
         return FunctionMetadata.merge(
             FunctionMetadata.from_function(
                 cls.run,
                 omit={'self', 'dependency_results'}),
             cls.get_shortlog_checks_metadata(),
-            cls.get_body_checks_metadata())
+            cls.get_body_checks_metadata(),
+            cls.get_issue_checks_metadata())
+
+    @staticmethod
+    def get_host_from_remotes():
+        """
+        Retrieve the first host from the list of git remotes.
+        """
+        remotes, _ = run_shell_command(
+            'git config --get-regex \'^remote.*.url$\'')
```

Comment:
rather use double quotes so you don't need to escape the single quotes:
```
"git config --get-regex '^remote.*.url$'"
```
btw if you want to improve performance, you can pre-split the command (up to you):
```
('git', 'config', '--get-regex', '^remote.*.url$')
```